test/kdump_test.rb

```diff
@@ -159,4 +167,290 @@
       end
     end
   end
+
+  describe ".ReadKdumpKernelParam" do
+    before do
+      allow(Yast::Bootloader).to receive(:kernel_param).and_return kernel_param
+      Yast::Kdump.ReadKdumpKernelParam
+    end
+
+    let(:is_crashkernel_set) { Yast::Kdump.crashkernel_param }
+    let(:crashkernel_needed) { Yast::Kdump.add_crashkernel_param }
+    let(:several_ranges) { Yast::Kdump.crashkernel_list_ranges }
+    let(:allocated_mem) { Yast::Kdump.allocated_memory }
+
+    context "when the param is not present" do
+      let(:kernel_param) { :missing }
+
+      it "reports param as not found" do
+        expect(is_crashkernel_set).to eq false
```

Comment:
I do not like this as it adds another redirection and if error happen it print this line. I think if it use whole `Yast::Kdump.crashkernel_param` it is more obvious what is wrong when test failed